app/models/user.rb

```diff
@@ -104,8 +104,8 @@ def student
 
   validates :name, :email, :country, :location, presence: true, unless: :github_import
 
-  accepts_nested_attributes_for :attendances, allow_destroy: true
-  accepts_nested_attributes_for :roles, allow_destroy: true
+  accepts_nested_attributes_for :attendances, allow_destroy: true, limit: 3
+  accepts_nested_attributes_for :roles
```

Comment:
Hi @beanieboi Thanks for your review!
The removal is not in my branch locally. 
Can it be that this happens with merging the master into the branch? IIRC I added the `allow_destroy: true` for roles in another PR. So can it be that this PR overrides the other one?